docs/model-configuration.md

```diff
@@ -54,6 +54,7 @@ The list features only model wrappers which intoduce new configuration values in
 1. `confidence_threshold`: float - probability threshold value for bounding box filtering
 1. `labels`: List - list of class labels
 1. `path_to_labels`: str - path to file with labels. Overrides the labels, if they sets via `labels` parameter
+1. `postproc_masks`: bool - resize and apply 0.5 threshold to instance segmentation masks
```

Comment:
I would not make the name shorter and keep `postprocess_semantic_masks`

Reply:
renamed